tests/standalone/ClickUsing.py

```diff
@@ -0,0 +1,16 @@
+import click
+
+# nuitka-skip-unless-imports: click
+
+@click.command()
+#@click.option("--count", default = 1, help = "Number of greetings.")
```

Comment:
We should ideally have some options configured. Any specific reason to comment them out?

Reply:
Those options require that the user input his/her name and the value of count.When I ran it that way, Travis CI failed with an error that indicated that no output was received: since "name" and "count" had not been set.
Hence, I went ahead to put the values of "name" and "count" in the test such that the hello() function will have its desired parameters.

Reply:
May I suggest that you enhance the test runner, to allow passing of arguments. You could change it to using `optparse` (or do you want to try click?) and instead of making sys.argv manual `in` tests, it would e.g. accept --with-command-args="...." and that would be used in `run_all.y`.

@tab1tha Do you know what I am talking about, I mean the binary behind / called in `nuitka.tools.testing.Common.compareWithCPython`. Do you think you can do this? It's actually a shame.

I guess you would just convert the "extra flags" in that function to proper command line options of the tool, and not have to change much other uses at all.

That way, we would test click properly, as well as command line handling for standalone binaries itself. I don't think anything really covers the correctness and compatibility of `sys.argv` in Nuitka for that matter.

Reply:
Thanks for the suggestion @kayhayen . When I read further into click's documentation, I found out that it has its own runner!. It is called CliRunner and it invokes commands as command line scripts. I used it, in the new commit and it works . Is this satisfactory?

Reply:
@tab1tha I don't think it is useful to use `clirunner` in the test.. We would like to see you manage to improve and extend the test runner in the described way. Do you think you can do it? If you want to use `click` instead of `optparse` for the argument handling in the Nuitka test runner, fine, but we would need to see it.